### ISSUE TITLE
Feature request: Set default output file extension in "Preferences" or in "Project Settings"

### ISSUE THREAD

**Issue by jofralogo** [bdkjones/CodeKit-1-Old#586]
It would be great if we could change the default output file extension in preferences, not just the folder. For example, if you work with HAML and you want to use .php extension instead of .html.

At the moment, this is my "workflow" to deal with that:
1) Save my haml file
2) Delete generated .html file
3) Go to the 'Files' window and select the haml file (or files)
4) Click right mouse button, select "Set output paths...", and write "*.php"
5) Compile them again.

I have to do the same every time I add a new file to the project... And it's totally anoying.

Maybe there is a better way to do that but I didn't find it yet.

BTW, I <3 Codekit. I don't know how I could live without it!

**Comment by bdkjones** [comment]
There is a better way. Just find the output file in the Finder and rename it from "whatever.html" to "whatever.php". CodeKit will pick up the change and set the new output path automatically.

**Comment by jofralogo** [comment]
I agree with you, your way is better, but it is still a little bit annoying (specially when adding new files to the project)

If it were possible, I would rather a "Preferences" way to do that...

Thanks for your answer!
